--- pass_import/__init__.py.bak	2023-01-17 14:37:43.436764500 +0000
+++ pass_import/__init__.py	2023-01-17 14:37:56.019979498 +0000
@@ -19,6 +19,19 @@
     '__email__', '__license__', '__copyright__'
 ]
 
+__title__ = 'pass-import'
+__summary__ = ('A pass extension for importing data from most of '
+               'the existing password manager.')
+__uri__ = 'https://github.com/roddhjav/pass-import'
+
+__version__ = '3.4'
+
+__author__ = 'Alexandre Pujol'
+__email__ = 'alexandre@pujol.io'
+
+__license__ = 'GPL3'
+__copyright__ = f'Copyright 2017-2020 {__author__}'
+
 
 class ManagerError(Exception):
     """Errors related to managers' management. Most likely a bug if raised."""
